elements/pfe-card/test/pfe-card.spec.ts

```diff
@@ -80,302 +56,142 @@ describe('<pfe-card>', function() {
     expect(cardFooterSlot.assignedNodes().length).to.equal(1);
   });
 
-  it(`should render a header and footer when content for those deprecated slots are added dynamically`, async function() {
-    const element = await createFixture<PfeCard>(dynamicHeaderFooter);
-
-    const header = document.createElement('h2');
-    header.setAttribute('slot', 'pfe-card--header');
-    header.textContent = 'Card Header';
+  describe('size', () => {
+    describe('is unset', () => {
+      it('should have default sizing for card parts', async function() {
+        const element = await createFixture<PfeCard>(TEMPLATES.card1);
+        const cardHeader = element.shadowRoot!.querySelector<HTMLSlotElement>('[part="header"]')!;
```

Comment:
it's better to avoid testing based on shadow dom structure. can you check the computed size of the whole element or it's slotted content instead?